lib/provider/camunda/element-templates/parts/DescriptionProps.js

```diff
@@ -164,4 +175,30 @@ function getEventDefinitionType(businessObject) {
   }
 
   return eventDefinition.$type;
+}
+
+function createTemplateNotFoundEntry(element, modeling, translate) {
+  var templateId = getTemplateId(element);
+
+  var html = domify(
+    '<p>' +
+    translate(
+      'The applied Template <{templateId}> was not found and therefore its data is not accessible.' +
+      '<br />' +
+      '<br />' +
+      'Unlink in order to access the data.', { templateId: templateId }
+    ) +
+    '</p>' +
+    '<p style="text-align: right;">' +
+    '<a class="bpp-entry-link bpp-entry-link-button">Unlink</a>' +
+    '</p>'
+  );
+
+  domQuery('.bpp-entry-link', html).addEventListener('click', function() {

```

Comment:
I like that we don't try to handle this via `data-action` (if it's possible with that at all) 😄  That's fine for me.

Reply:
At least we can avoid the `set` hack 👍